types/theo/theo-tests.ts

```diff
@@ -0,0 +1,39 @@
+import { Theo } from "theo";
```

Comment:
This doesn't seem like it's actually testing the usage pattern of the library.

In your test file, you'll want to have something close to an actual usage example, so for example:
```ts
import theo = require('theo');
theo.convert( // etc. etc.
```

If you copy and paste the usage example from https://www.npmjs.com/package/theo it doesn't quite work with your current typings, but with my suggested modifications I think it will.

Reply:
Thank you for the advice! Updates made 👍